server/src/controllers/Events.js

```diff
@@ -65,10 +69,9 @@ module.exports = {
           public_id: image.public_id
         };
       }
-      const event = await Event.findByIdAndUpdate(eventId, req.body);
-      res.json({
-        id: event._id
-      });
+      let event = await Event.updateOne({_id: mongoose.Types.ObjectId(eventId)}, req.body);
+      event = await Event.findById(eventId);
```

Comment:
Any special reasons for changing the update function here ? Both look the same to me tbh.

Reply:
It was not giving proper output so had to break it into two different steps.